hearts/tests/test_hearts.py

```diff
@@ -163,18 +163,19 @@ def test_is_valid_follow():
 
 
 def test_lead_the_trick():
-    test_round = Round(PLAYER_LIST)
-    fake_hand = Hand.deserialize(['7d', '6h', 'Ah', '2s'])
-    fake_trick = Trick([(P1, Card('2', 'c'))])
-    test_round.hands[P4] = fake_hand
-    test_round.turn_counter = 3
-    test_round.tricks.append(fake_trick)
-
-    test_round.lead_the_trick(P4, Card('2', 's'))
-    assert test_round.tricks[-1] == Trick([(P4, Card('2', 's'))])
+    round1 = Round(PLAYER_LIST)
+    hand = Hand.deserialize(['7d', '6h', 'Ah', '2s'])
+    trick = Trick([(P1, Card('2', 'c'))])
+    round1.hands[P4] = hand
+    round1.turn_counter = 3
+    round1.tricks.append(trick)
+
+    round1.lead_the_trick(P4, Card('2', 's'))
```

Comment:
nitpick: if the most recent trick is not yet complete (in this test, it only contains one card), should this raise an exception? Or should we just assume that `lead_the_trick` is never called if the previous trick has not yet completed?